src/rtc/rtc_s1.py

```diff
@@ -499,7 +540,7 @@ def run_parallel(cfg: RunConfig, logfile_path, flag_logger_full_format):
     # Execute the single burst processes using multiprocessing
     with multiprocessing.Pool(num_workers) as p:
         processing_result_list =\
-            p.starmap(process_child_runconfig,
+            p.starmap(process_child_runconfig_subprocess,
```

Comment:
`process_child_runconfig()` is still in the code. That function eventually needs to be replaced by the new function `process_child_runconfig_subprocess()`